composeApp/build.gradle.kts

```diff
@@ -88,7 +88,7 @@ kotlin {
     iosSimulatorArm64()
 
     cocoapods {
-        ios.deploymentTarget = "12.0"
+        ios.deploymentTarget = "15.3"
```

Comment:
If sentry is the only thing forcing us to jump versions, can we use an older version of sentry?

Reply:
This was me during bug fixing. But I just copied the minimum we had on the iOS config files. I think we had a lower minimum here than there, so I just matched it. Our minimum is 15.3 right?

Reply:
its `13.0`

Reply:
Oh, then I have to fix it on a bunch of different places.

Reply:
Addressed in https://github.com/ooni/probe-multiplatform/pull/133